### ISSUE TITLE
Upgrading to 3.1.3 appears to be breaking nested <Choose>

### ISSUE THREAD

**Issue by ZeeStorm** [AlexGilleran/jsx-control-statements#52]
This seems to occur when I put a `<Choose>` directly inside of a `<When>` when there are no other components. It may occur in many other cases as well. Reverting to 3.1.2 fixed my issue. I can provide the whole code file if requested via email.

Thanks :D

Stack:
```
build modulesModuleBuildError: Module build failed: TypeError: /ephemeral0/work/2a4d4e3236001935/src/transpile/components/Datasets/View/List/MenuCell.js: Cannot read property 'attributes' of undefined
   at exports.addKeyAttribute (/home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/util/ast.js:86:22)
   at Object.exports.getSanitizedExpressionForContent (/home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/util/ast.js:113:7)
   at /home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/chooseStatement.js:30:49
   at Array.reduceRight (native)
   at getBlocks (/home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/chooseStatement.js:17:25)
   at /home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/chooseStatement.js:65:18
   at PluginPass.visitor.JSXElement (/home/compellon/BuildAgent/work/2a4d4e3236001935/node_modules/jsx-control-statements/src/index.js:19:26)
...
```

Code snippet:
```
<Choose>
    <When condition={ dataset.status === 'COMPELTE' }>
        <Choose>
            <When condition={ preferences.app_mode }>
                ...
            </When>
            <Otherwise>
                ...
            </Otherwise>
        </Choose>
    </When>
    <Otherwise>
        ...
    </Otherwise>
</Choose>
```

**Comment by AlexGilleran** [comment]
So I can only reproduce this if I put `key` on the `<Choose>` statement, is that something that you've got but took out? 

If you want to send me the code at alex -at- alexgilleran.com I'd be glad to look at it :).

**Comment by AlexGilleran** [comment]
Ok so I've just published v3.1.4 which should fix this - can you try it out @ZeeStorm and @thatjsguy ?

**Comment by AlexGilleran** [comment]
@thatjsguy confirmed it worked for him, works for me too, closing. Reopen if it doesn't work for you @ZeeStorm :).
